src/server/services/procedures/alpha-vantage/alpha-vantage.js

```diff
@@ -79,14 +79,17 @@ AlphaVantage.equitiesSymbolSearch = async function(search) {
     return matches;
 };
 
-AlphaVantage._rawSearchEquities = async function(apifunction, symbol, interval, attributes, dateFormat) {
+AlphaVantage._rawSearchEquities = async function(apifunction, symbol, interval, attributes, dateFormat, startDate, endDate) {
+    startDate = startDate ? +startDate : -Infinity;
```

Comment:
Can this be something other than a number? If so, we need to check that it isn't `NaN`. It probably isn't a problem since the RPC arguments should all be type checked but I actually didn't expect to see the coercion to a number (since they should all be numbers, right?)

Reply:
@brollb This one's type `Number=`, but we omit an actual default value, so it's either a number or undefined, which is used for "no filter". If nan were provided, it'd be falsy so we'd get `-Infinity` (no filtering).

Reply:
Ah, I hadn't ever used `+` to convert dates to unix timestamps before. Makes sense.